templates/issue.html

```diff
@@ -292,41 +300,29 @@
 					<input name="issue_number" value="{{.IssueNumber}}" hidden></input>
 					<input name="repo_id" value="{{.RepoId}}" hidden></input>
 					{{if lt $.PinnedIssuesCount 3}}
-					<button type="submit" > Pin Issue </button>
+					<button type="submit" ><i class="fas fa-thumbtack"></i> Pin Issue </button>
 					{{end}}
 				</form>
 				{{end}}
 				{{if eq .Pinned "Pinned"}}
 				<form method="post" action="/{{$.UserName}}/{{$.RepoName}}/issues/{{.IssueNumber}}/unpin">
 					<input name="issue_id" value="{{.ID}}" hidden></input>
 					<input name="issue_number" value="{{.IssueNumber}}" hidden></input>
```

Comment:
issue number is unnecessary. Just get it from params

Reply:
done